src/backend/InvenTree/InvenTree/unit_test.py

```diff
@@ -372,25 +372,46 @@ def assertNumQueriesLessThan(
         self.assertLess(n, value, msg=msg)
 
 
-class InvenTreeTestCase(ExchangeRateMixin, UserMixin, TestCase):
-    """Testcase with user setup build in."""
-
-
-class InvenTreeAPITestCase(ExchangeRateMixin, TestQueryMixin, UserMixin, APITestCase):
-    """Base class for running InvenTree API tests."""
+class PluginRegistryMixin:
```

Comment:
Would it be possible to flag this to ensure it only introduces a performance hit if plugins are required for tests?